web/src/components/troubleshoot/SupportBundleList.jsx

```diff
@@ -67,6 +93,63 @@ class SupportBundleList extends React.Component {
         });
       });
   };
+  pollForBundleAnalysisProgress = async () => {
+    const { newBundleSlug } = this.props;
```

Comment:
I don't want to add this polling but in order to display the progress bar adding this is necessary. Will refactor in the future.